auto_nag/scripts/patch_closed_bug.py

```diff
@@ -103,7 +103,7 @@ def get_bz_params(self, date):
     def get_autofix_change(self):
         return {
             "comment": {
-                "body": "A patch has been attached on this bug, which was already closed. Filing a separate bug will ensure better tracking. If this was not by mistake and further action is needed, please alert the appropriate party."
+                "body": "A patch has been attached on this bug, which was already closed. If the patch doesn't change behavior (e.g. landing a test case, or fixing a typo), there's nothing to do. Otherwise, filing a separate bug will ensure better tracking. If this was not by mistake and further action is needed, please alert the appropriate party."
```

Comment:
Thanks for rewording this!

Minor nit: "There's nothing to do" is a bit vague & might be misinterpreted.  (in particular, in the "fine" scenarios, there's not "nothing to do" but in fact at least 1-2 more things to do: land the followup patch, possibly after getting it reviewed)

Maybe better to just add this clarification as a parenthetical afterthought, with "feel free to disregard this message" instead of "there's nothing to do"?  Something like:
> A patch has been attached on this bug, which was already closed. Filing a separate bug will ensure better tracking. If this was not by mistake and further action is needed, please alert the appropriate party. (Or: if the patch doesn't change behavior -- e.g. landing a test case, or fixing a typo -- then feel free to disregard this message.)

Reply:
Done in https://github.com/mozilla/relman-auto-nag/pull/1549